packages/authentication/src/definitions.ts

```diff
@@ -114,6 +114,14 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.1.0
    */
   getIdToken(options?: GetIdTokenOptions): Promise<GetIdTokenResult>;
+  /**
+   * Returns a deserialized JSON Web Token (JWT) used to identify the user to a Firebase service.
+   *
+   * @since 6.4.0
```

Comment:
```suggestion
   * @since 7.4.0
```

You need to update those version numbers. Next minor release will be `7.4.0`.